src/Form/CaptchaPointDeleteForm.php

```diff
@@ -41,7 +41,7 @@ public function getConfirmText() {
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $this->entity->delete();
-    drupal_set_message($this->t('Captcha point %label has been deleted.', array('%label' => $this->entity->label())));
+    drupal_set_message($this->t('Captcha point %label has been deleted.', array('%label' => $this->entity->getOriginalId())));
     $form_state->setRedirectUrl($this->getCancelUrl());
```

Comment:
Didn't work == ''

label() returned an empty string, I could have just removed the name but instead replaced it with the form machine name so it is kind of useful again.